js/main.js

```diff
@@ -40,6 +48,37 @@ Site = {
 
 };
 
+Site.Shop = {
+  init: function() {
+    var _this = this;
+
+    this.Search.init();
+  },
+
+  Search: {
+    init: function() {
+      var _this = this;
+
+      if ($('#searchform').length) {
+        _this.bindFocus();
+      }
+    },
+
+    bindFocus: function() {
```

Comment:
what does this do? adds an active class to the form when you click on anything? is that just for styling? I don't see anything different on my WP and cant see any classes for the active. And the active is never removed?